Node API/node-rest-template/test/web/movies.router.spec.js

```diff
@@ -0,0 +1,47 @@
+const request = require('supertest');
+const { initializeApp } = require('../../lib');
+
+const errorCodeToStatusMap = require('../../src/config/errorCodeToStatusMap');
+const { Movie } = require('../../src/lib/movies/movies.model');
+const moviesRouter = require('../../src/web/movies.router');
+const { movie1, movie2, movie3 } = require('../fixtures/movies');
+
+const movies = [movie1, movie2, movie3];
+
+describe('movies/router', () => {
+  const app = initializeApp({ router: moviesRouter, errorCodeToStatusMap });
+  const server = app.listen();
+
+  const serviceResponse = {
+    movies: movies.map((movie) => {
+      return { ...movie, release_date: movie.release_date.toISOString() };
+    }),
+  };
+
+  afterAll(() => {
+    server.close();
+  });
+
+  describe('GET /', () => {
+    beforeEach(async () => {
+      await Movie.query().del();
+      await Movie.query().insert(movies);
+    });
+
+    it('should return list of movies', async () => {
```

Comment:
There are missing tests in this suite, for example:
- `when a query is passed`
- `when filter parameter is passed` (subtest of the first item)
-  `when sort parameter is passed` (subtest of the first item)